src/consensus/proof.rs

```diff
@@ -6,6 +6,7 @@
 // KIND, either express or implied. Please review the Licences for the specific language governing
 // permissions and limitations relating to use of the SAFE Network Software.
 
+use serde::Serialize as SerdeSerialize;
```

Comment:
Why not just `use serde::Serialize` ?

Reply:
avoid confliction with std::Serialize

Reply:
What is `std::Serialize` ?

Reply:
Do you mean `derive(Serialize)`? If yes, then that is not `std` but also `serde` and its a different namespace (trait vs proc-macro) so it doesn't conflict.

Reply:
ah, my mistake, will change it